src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -61,17 +65,19 @@ static void LaunchAndWait(Permutation permutation)
             var sessionIdArgument = string.Format(" --sessionId={0}", SessionId);
 
             var exe = new FileInfo(permutation.Exe);
+            var arguments = PermutationParser.ToArgs(permutation) + processIdArgument + sessionIdArgument;
 
-            var pi = new ProcessStartInfo(exe.FullName, PermutationParser.ToArgs(permutation) + processIdArgument + sessionIdArgument)
+            var pi = new ProcessStartInfo(exe.FullName)
             {
+                Arguments = arguments,
```

Comment:
Don't use the arguments property while there is constructor overload that accepts arguments.